sdk/storage/azure-storage-blob/azure/storage/blob/_shared/parser.py

```diff
@@ -18,3 +18,10 @@ def _str(value):
 
 def _to_utc_datetime(value):
     return value.strftime('%Y-%m-%dT%H:%M:%SZ')
+
+
+def get_empty_chunk(chunk_size):
+    empty_chunk = b''
+    for i in range(0, chunk_size):  # pylint:disable=unused-variable
```

Comment:
nit: 
you can do `for _ in range(0, chunk_size):` and avoid disabling pylint.
it's also a normal convention in python if you are not using the variable